src/components/molecules/TextView.tsx

```diff
@@ -0,0 +1,159 @@
+import React, { FC } from 'react';
+import { IWidgetCountBySeverityTextData } from '../../models/WidgetData';
+import { Typography } from '../atoms';
+import { Theme, makeStyles } from '@material-ui/core';
+import { roadIconColors } from '../../style';
+import { useTranslation } from 'react-i18next';
+import Person from '../../assets/Person.png';
+import Ambulance from '../../assets/Ambulance.png';
+import Crutches from '../../assets/Crutches.png';
+import Box from '@material-ui/core/Box';
+
+interface IProps {
+  data: IWidgetCountBySeverityTextData;
+  segmentText: string;
+}
+interface AProps {
+  accidentsCount: Number;
+}
+
+const red = roadIconColors.red;
+
+const useStyles = makeStyles((theme: Theme) => ({
+  root: {
+    display: 'flex',
+    flexDirection: 'column',
+    height: '100%',
+    justifyContent: 'space-evenly',
+    letterSpacing: 1,
+    padding: `0 ${theme.spacing(6)}px`,
```

Comment:
use spacing like this:
`theme.spacing(0, 6),`
see
https://material-ui.com/customization/spacing/